src/Umbraco.PublishedCache.HybridCache/Persistence/DatabaseCacheRepository.cs

```diff
@@ -429,32 +429,46 @@ private void RebuildMemberDbCache(IContentCacheDataSerializer serializer, int gr
             IEnumerable<IMember> descendants =
                 _memberRepository.GetPage(query, pageIndex++, groupSize, out total, null, Ordering.By("Path"));
             ContentNuDto[] items = descendants.Select(m => GetDtoFromContent(m, false, serializer)).ToArray();
-            Database.BulkInsertRecords(items);
+            _ = Database.BulkInsertRecords(items);
             processed += items.Length;
         }
         while (processed < total);
     }
 
-    private void DeleteForObjectType(Guid nodeObjectType) =>
-        Database.Execute(
-            @"
-                DELETE FROM cmsContentNu
-                WHERE cmsContentNu.nodeId IN (
-                    SELECT id FROM umbracoNode WHERE umbracoNode.nodeObjectType = @objType
-                )",
-            new { objType = nodeObjectType });
-
-    private void DeleteForObjectTypeAndContentTypes(Guid nodeObjectType, IReadOnlyCollection<int> contentTypeIds) =>
-        Database.Execute(
-            $@"
-                DELETE FROM cmsContentNu
-                WHERE cmsContentNu.nodeId IN (
-                    SELECT id FROM umbracoNode
-                    JOIN {Constants.DatabaseSchema.Tables.Content} ON {Constants.DatabaseSchema.Tables.Content}.nodeId=umbracoNode.id
-                    WHERE umbracoNode.nodeObjectType = @objType
-                    AND {Constants.DatabaseSchema.Tables.Content}.contentTypeId IN (@ctypes)
-                )",
-            new { objType = nodeObjectType, ctypes = contentTypeIds });
+    private void RemoveByObjectType(Guid objectType, IReadOnlyCollection<int>? contentTypeIds)
+    {
+        // remove all - if anything fails the transaction will rollback
```

Comment:
```suggestion
        // If the provided contentTypeIds collection is empty, remove all records for the provided object type.
```

Let's update these comments to better describe what's going on in this method.

Reply:
Done